test/check-application

```diff
@@ -1360,8 +1360,8 @@ class TestApplication(testlib.MachineCase):
         b.set_input_text('#run-image-dialog-env-2-value', 'GRAPE')
         b.click('#run-image-dialog-env-2-btn-close')
         b.click('.env-form .btn-add')
-        b.set_input_text('#run-image-dialog-env-2-key', 'RHUBARB')
-        b.set_input_text('#run-image-dialog-env-2-value', 'STRAWBERRY')
+        # Test inputting an key=var entry
+        b.set_input_text('#run-image-dialog-env-2-value', "RHUBARB=STRAWBERRY", value_check=False)
```

Comment:
Can you please test it with more values at the same time? To test the "bulk" part as well.

Reply:
Sure, done.